app/javascript/add-another.js

```diff
@@ -71,11 +71,32 @@ const setUpSuggestions = (section) => {
 
 const remove = (section) => {
   const sectionList = section.closest('[data-add-another-role="sectionList"]')
+  showItemRemovedFeedback(section);
   section.remove();
   setUpSections(sectionList); // Some JS triggers, in particular radio conditional reveals, must be re-initialised
   setUpAddButton(sectionList.closest('[data-module="add-another"]'));
 }
 
+const showItemRemovedFeedback = (section) => {
+  const topLevelElement = section.closest('[data-module="add-another"]');
+  const feedback = document.createElement("div");
+  feedback.className = "add-another-removed-feedback";
+
+  const text = document.createElement("div");
+  text.className = "add-another-removed-feedback-text govuk-body";
+  text.innerHTML = topLevelElement.dataset.addAnotherRemovedFeedbackText;
```

Comment:
ok - I know see that this bit `addAnotherRemovedFeedbackText`, is being told what text to show inside the feedback element box thing.